ProcessLib/TwoPhaseFlowWithPP/CMakeLists.txt

```diff
@@ -1,26 +1,46 @@
 AddTest(
-    NAME 2D_TwoPhase_PP_Lia_quad
+    NAME 2D_TwoPhase_PP_Lia_quad_1
     PATH Parabolic/TwoPhaseFlowPP/Liakopoulos
     EXECUTABLE ogs
-    EXECUTABLE_ARGS Twophase_Lia_quad2_short.prj
+    EXECUTABLE_ARGS TwoPhase_Lia_quad_short.prj
     TESTER vtkdiff
     REQUIREMENTS NOT OGS_USE_MPI
-    ABSTOL 1e-8 RELTOL 1e-12
+    ABSTOL 1e-2 RELTOL 1e-4
     DIFF_DATA
-    Lia_20.vtu twophaseflow_pcs_0_ts_119_t_20.000000.vtu capillary_pressure capillary_pressure
-    Lia_20.vtu twophaseflow_pcs_0_ts_119_t_20.000000.vtu gas_pressure gas_pressure
-    Lia_20.vtu twophaseflow_pcs_0_ts_119_t_20.000000.vtu saturation saturation
+    h2_Liako_20.vtu twophaseflow_pcs_0_ts_218_t_20.000000.vtu saturation saturation
 )
 AddTest(
-    NAME LARGE_2D_TwoPhase_PP_Lia_quad
+    NAME 2D_TwoPhase_PP_Lia_quad_2
     PATH Parabolic/TwoPhaseFlowPP/Liakopoulos
     EXECUTABLE ogs
-    EXECUTABLE_ARGS Twophase_Lia_quad2_large.prj
+    EXECUTABLE_ARGS TwoPhase_Lia_quad_short.prj
     TESTER vtkdiff
     REQUIREMENTS NOT OGS_USE_MPI
-    ABSTOL 1e-8 RELTOL 1e-12
+    ABSTOL 20 RELTOL 1e-3
```

Comment:
In the previous test, the results are self-comparison, therefore the difference can be very small. Currently the comparisons are made between ogs5 and ogs6, those to some extent explain why tolerances degrades.
To be sure, I will make another test